doc/geany.txt

```diff
@@ -2776,14 +2776,35 @@ The project menu items are detailed below.
 New project
 ^^^^^^^^^^^
 
-To create a new project, fill in the *Name* field. By default this
-will setup a new project file ``~/projects/name.geany``. Usually it's
-best to store all your project files in the same directory (they are
-independent of any source directory trees).
-
-The Base path text field is setup to use ``~/projects/name``. This
-can safely be set to any existing path -- it will not touch the file
-structure contained in it.
+There are two ways of creating new projects, either by using
+*Project->New* menu item or by using *Project->New from Folder* menu
+item.
+
+New
+    This method is more suitable for creating new, empty projects from
+    scratch at the default location without having any existing sources.
+
+    To create a new project, fill in the *Name* field. By default this
+    will setup a new project file ``~/projects/name/name.geany``.
+
+    The Base path text field is setup to use ``~/projects/name``. This
+    can safely be set to any existing path -- it will not touch the file
+    structure contained in it.
```

Comment:
But it will add name.geany file there.

Reply:
I basically just kept the previous text here but I think it's kind of OK - the project filename is editable and can be changed to a different path and it should be clear where that file gets created by checking the Filename field.